app/queries/articles/active_threads_query.rb

```diff
@@ -19,12 +19,14 @@ def self.call(relation: Article.published, **options)
                  elsif time_ago
                    relation = relation.where(published_at: time_ago.., score: MINIMUM_SCORE..).presence || relation
                    relation.order(comments_count: :desc)
+                 elsif tags.present?
+                   relation = tags.size == 1 ? relation.cached_tagged_with(tags.first) : relation.tagged_with(tags)
+                   relation = relation.where(published_at: 5.days.ago.., score: MINIMUM_SCORE..).presence || relation
+                   relation.order("last_comment_at DESC NULLS LAST")
                  else
-                   relation = relation.where(published_at: (tags.present? ? 5 : 2).days.ago..,
-                                             score: MINIMUM_SCORE..).presence || relation
+                   relation = relation.where(published_at: 2.days.ago.., score: MINIMUM_SCORE..).presence || relation
                    relation.order("last_comment_at DESC NULLS LAST")
                  end
-      relation = tags.size == 1 ? relation.cached_tagged_with(tags.first) : relation.tagged_with(tags)
```

Comment:
This line was causing the bugginess with sidebar tags on the Forem Fleet. To quote @mstruve: 

> That will often return nothing bc even though people post often, posting for a specific tag is much less common.